packages/cloud-cognitive/src/components/Saving/Saving.js

```diff
@@ -59,24 +59,21 @@ export let Saving = ({
   };
 
   const statusObj = getStatusObj();
-
-  const classnames = cx(`${pkg.prefix}-saving`, {
-    [className]: className,
-  });
+  const blockClass = `${pkg.prefix}--saving`;
 
   return (
-    <div className={classnames}>
+    <div className={cx(blockClass, className)}>
```

Comment:
Format of selector should be `prefix--Block__Element--Modifier`

Reply:
_i.e._ `${blockClass}__message`, `${blockClass}__error-icon`, `${blockClass}__text`, `${blockClass}__buttons`.

Reply:
gotcha